dbdialog/src/org/pentaho/ui/database/DatabaseConnectionDialog.java

```diff
@@ -43,10 +43,9 @@ public void registerClass( String key, String className ) {
     extendedClasses.put( key, className );
   }
 
-  public XulDomContainer getSwtInstance( Shell shell ) throws XulException {
+  public XulDomContainer getSwtInstance( SwtXulLoader loader, Shell shell ) throws XulException {
```

Comment:
@AlexanderBuloichik So will it be a safe idea to preserve the original signature and pass a new SwtXulLoader() to the new one?